platform/event/tss_message_transaction.proto

```diff
@@ -28,37 +28,51 @@ package com.hedera.hapi.platform.event;
  * limitations under the License.
  */
 
-option java_package = "com.hedera.hapi.platform.event.legacy";
-// <<<pbj.java_package = "com.hedera.hapi.platform.event">>> This comment is special code for setting PBJ Compiler java package
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.tss">>> This comment is special code for setting PBJ Compiler java package
```

Comment:
We very much should not be changing these packages (particularly as this change does not match the folder, which breaks some SDKs).